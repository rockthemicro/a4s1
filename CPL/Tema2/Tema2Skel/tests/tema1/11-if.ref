Semantic error: No method main in class Main
Compilation halted
