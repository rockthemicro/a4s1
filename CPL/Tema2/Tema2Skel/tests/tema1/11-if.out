line 1:0 token recognition error at: 'c'
line 1:1 token recognition error at: 'l'
line 1:2 token recognition error at: 'a'
line 1:3 token recognition error at: 's'
line 1:4 token recognition error at: 's'
line 1:6 token recognition error at: 'A'
line 1:8 token recognition error at: '{'
line 2:4 token recognition error at: 'f'
line 2:5 token recognition error at: '('
line 2:6 token recognition error at: 'x'
line 2:8 token recognition error at: ':'
line 2:10 token recognition error at: 'I'
line 2:11 token recognition error at: 'n'
line 2:12 token recognition error at: 't'
line 2:13 token recognition error at: ')'
line 2:15 token recognition error at: ':'
line 2:17 token recognition error at: 'I'
line 2:18 token recognition error at: 'n'
line 2:19 token recognition error at: 't'
line 2:21 token recognition error at: '{'
line 3:8 token recognition error at: 'i'
line 3:9 token recognition error at: 'f'
line 3:11 token recognition error at: 'x'
line 3:13 token recognition error at: '<'
line 3:14 token recognition error at: '='
line 3:16 token recognition error at: '5'
line 3:18 token recognition error at: 't'
line 3:19 token recognition error at: 'h'
line 3:20 token recognition error at: 'e'
line 3:21 token recognition error at: 'n'
line 3:23 token recognition error at: 'x'
line 3:25 token recognition error at: 'e'
line 3:26 token recognition error at: 'l'
line 3:27 token recognition error at: 's'
line 3:28 token recognition error at: 'e'
line 3:30 token recognition error at: 'x'
line 3:32 token recognition error at: '+'
line 3:34 token recognition error at: '1'
line 3:36 token recognition error at: 'f'
line 3:37 token recognition error at: 'i'
line 4:4 token recognition error at: '}'
line 4:5 token recognition error at: ';'
line 5:0 token recognition error at: '}'
line 5:1 token recognition error at: ';'
